assignment.md

```diff
@@ -12,46 +12,83 @@ If the version control concepts or `git` commands are still confusing to you, co
 
     ```bash
     # git command used and new list of branches
+    The git command i used was: git checkout -b submission and the branches that were returned were main and submission.
     ```
 
  1. List the current `remote` name and hyperlink destination of this assignment repository (Hint: if you are not sure, add `--help` after your command to check remotes.). 
 
     ```bash
     # your git command and output here
+     git remote -v
+origin  https://github.com/emmamidori/version_control_assignment.git (fetch)
     ```
 
 
  2. Add a second remote destination named `upstream` to your assignment repository, `upstream` is a common name used for the repository that people have forked from, in our case, the `upstream` will be at `https://github.com/WCM-datascibasics/version_control_assignment`. List the new list of remotes after you've added the new remote. 
  
     ```bash
     # list the new list of remotes here
+    origin
```

Comment:
a `git remote -v` output would've been nice, so I can see the `.git` urls, but this is fine too.